website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Connecting your GitLab Account"
```

Comment:
This is kind of hard to grok in the sidebar - what do you think about renaming this page and the GitHub page?

<img width="278" alt="Screen Shot 2020-10-02 at 6 08 33 PM" src="https://user-images.githubusercontent.com/1541139/94973705-5f1c3d80-04da-11eb-97af-75ecc1c453f6.png">

I was thinking something like `Connecting GitLab` and `Connecting GitHub`.... but I'm not sure that that's any better. What do you think?

Reply:
Yeah, agree here - can do that here!

Reply:
I like this! Maybe if the provider list grows we'll want to nest it or some other better UX, but I think this is good for now!